src/sass/base/_reset.scss

```diff
@@ -1,3 +1,4 @@
+// sass-lint:disable single-line-per-selector
```

Comment:
We *could* disable the linter for the entire file, since this is a copy-paste CSS reset (I think from [here](https://meyerweb.com/eric/tools/css/reset/)). I don't have a strong opinion on that though :)